syft/workers/base.py

```diff
@@ -467,6 +467,28 @@ def execute_command(self, message: tuple) -> PointerTensor:
                 new_ids = return_id_provider.get_recorded_ids()
                 raise ResponseSignatureError(new_ids)
 
+    def execute_worker_function(self, message: tuple):
+        """Executes commands received from other workers.
+
+                Args:
+                    message: A tuple specifying the command and the args.
+
+                Returns:
+                    A pointer to the result.
+                """
+        command_name, (args, kwargs, return_ids) = message
+
+        command_name = command_name
+        response = getattr(self, command_name)(*args, **kwargs)
+        #  TODO [midokura-silvia]: send the tensor directly
+        #  TODO this code is currently necessary for the async_fit method in websocket_client.py
+        import torch
+
+        if isinstance(response, torch.Tensor):
```

Comment:
You can put isinstance syft.FrameworkTensor I think here and not have to import torch